app/controllers/additional_expenses_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AdditionalExpensesController < ApplicationController
+  def create
+    @additional_expense = AdditionalExpense.new(additional_expense_params)
+    authorize @additional_expense
+
+    respond_to do |format|
+      if @additional_expense.save
+        format.json { render json: @additional_expense.as_json, status: :created }
+      else
+        format.json { render json: @additional_expense.errors.as_json, status: :unprocessable_entity }
```

Comment:
The `.as_json` shouldn't be necessary here.  `render json: ...` calls `.to_json` behind the curtain, and at some point in the full test suite, `.to_json` stops behaving as expected. I was getting results like `"<AdditionalExpense #OBJECTID ... >"`, a string representation of the object reference rather than the object converted to json.  Also a .to_json call in application.html.erb would create failures. 

It was really weird. I ran rspec bisect for an entire day, but couldn't pinpoint the issue. Maybe OJ or some gem?

Anyway, I don't think it's important, this works fine. Just wanted to document the strangeness.

Reply:
its from a rails 7.2 update https://github.com/rails/rails/pull/51640 that PR changed where ActiveSupport got pulled in so it messed up calls to `to_json`. 

I spent way too long debugging it.